build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/AndroidCompose.kt

```diff
@@ -42,6 +43,12 @@ internal fun Project.configureAndroidCompose(
         kotlinOptions {
             freeCompilerArgs = freeCompilerArgs + buildComposeMetricsParameters()
         }
+
+        dependencies {
+            val bom = libs.findLibrary("androidx-compose-bom").get()
+            add("implementation", platform(bom))
```

Comment:
❤️  it!

So by specifying the platform version here in the convention plugin, we can pull in Compose dependencies individually with the version taken from the bom, right? That's super cool!

Reply:
Yes! And if any module needs to override the version (e.g. wants to use an alpha version) it can just explicitly add that dependency and it will override what's provided in the BOM.

Reply:
Top work, thanks for doing this.